src/pcapi/local_providers/fnac/fnac_stocks_provider.py

```diff
@@ -15,4 +16,7 @@ def synchronize_fnac_venues_stocks() -> None:
     )
 
     for venue in venues:
-        synchronize_fnac_stocks.synchronize_venue_stocks_from_fnac(venue)
+        try:
+            synchronize_fnac_stocks.synchronize_venue_stocks_from_fnac(venue)
+        except Exception:
+            logger.exception("Could not synchronize stock of venue=%s", venue.id)
```

Comment:
Peut-être une petite remarque finalement. Est-ce que l'on pourrait ajouter le nom de l'exception dans le message ?
```suggestion
            logger.exception("Could not synchronize stock of venue=%s due to exception %s", venue.id, exception)
```
Pour pouvoir voir dans sentry s'il y a plusieurs exceptions distinctes plutôt que d'avoir une erreur dans laquelle il faut faire "Prev" Prev Prev prev pour vérifier si c'est bien N fois la même exception ou si on en a plusieurs différentes

Reply:
C'est ce qu'a proposé Cyril aussi. J'ai fait !